packages/babel-minify/README.md

```diff
@@ -0,0 +1,97 @@
+# Neutrino Minify Middleware
+
+`@neutrinojs/minify` is Neutrino middleware for minifying source code using
```

Comment:
`@neutrinojs/babel-minify`. I won't make any more comments for this, you can just do a find and replace for these.